runbot_travis2docker/models/runbot_branch.py

```diff
@@ -4,13 +4,33 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 import re
+import os
 
 import subprocess
 import requests
 
 from openerp import fields, models, api, tools
 
 
+def _ssh_keyscan(ssh):
```

Comment:
Use a global flag to know if this key was created previously or LRU_CACHE

Reply:
@moylop260 
This is better idea!